language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/toml/ballerinatoml/completion/BallerinaTomlCompletionTest.java

```diff
@@ -0,0 +1,198 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.langserver.toml.ballerinatoml.completion;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.reflect.TypeToken;
+import org.ballerinalang.langserver.commons.workspace.WorkspaceDocumentException;
+import org.ballerinalang.langserver.completion.util.CompletionTestUtil;
+import org.ballerinalang.langserver.util.FileUtils;
+import org.ballerinalang.langserver.util.TestUtil;
+import org.eclipse.lsp4j.CompletionItem;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.jsonrpc.Endpoint;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Completion Test Interface.
+ */
+public abstract class BallerinaTomlCompletionTest {
+
+    private Endpoint serviceEndpoint;
+
+    private final Path testRoot = FileUtils.RES_DIR.resolve("toml" + File.separator
+            + "ballerina_toml" + File.separator + "completion");
+
+    private final JsonParser parser = new JsonParser();
+
+    private final Gson gson = new Gson();
+
+    @BeforeClass
+    public void init() {
+        this.serviceEndpoint = TestUtil.initializeLanguageSever();
+    }
+
+    @Test(dataProvider = "completion-data-provider")
+    public void test(String config, String configPath) throws WorkspaceDocumentException, IOException {
+        String configJsonPath = testRoot + File.separator +
+                "config" + File.separator + configPath + File.separator + config;
+        JsonObject configJsonObject = FileUtils.fileContentAsObject(configJsonPath);
+
+        String response = getResponse(configJsonObject);
+        JsonObject json = parser.parse(response).getAsJsonObject();
+        Type collectionType = new TypeToken<List<CompletionItem>>() {
+        }.getType();
```

Comment:
This code is not properly formatted. Isn't it?

Reply:
Formatted properly